lib/base.js

```diff
@@ -213,7 +213,7 @@ module.exports = {
     "max-nested-callbacks": ["warn", 5],
     "max-params": ["warn", 7],
     "max-statements": ["warn", 40],
-    "new-cap": "warn",
+    "new-cap": ["warn", { capIsNewExceptions: ["$.Deferred"] }],
```

Comment:
@koba04 `$.Deferred()` is ok, but `jQuery.Deferred()` is not. Both are allowd in globals.
https://github.com/cybozu/eslint-config/blob/05422fec18c159ef9f31c16a1ab24915491336bb/globals/kintone.js#L4-L5

Is this better?

```suggestion
    "new-cap": ["warn", { capIsNewExceptions: ["Deferred"] }],
```

Reply:
@teppeis 
Thank you for your suggestion! I've updated the option and a test.
Sorry, I've missed that the suggestion is used GitHub's suggestion feature 🙏

Reply:
It's hard to notice :)